bin/check.sh

```diff
@@ -26,10 +36,9 @@ gometalinter --concurrency=${NUM_CPU} --enable-gc --deadline=300s --disable-all\
   --enable=varcheck\
   --enable=vet\
   --enable=vetshadow\
-  ./...
+  /go/src/istio.io/pilot/...
```

Comment:
that won't work on people's machine will it ? (my directory is ~/go no /go)

Reply:
oh right, sorry, I didn't see the docker run on line 11
is that image built each time ? (cached maybe but still) ? 
can we put the linters in some prow image we already have ?